packages/channel/test/insights-query-builder.test.ts

```diff
@@ -399,7 +345,7 @@ void describe('insights query builder tests', () => {
   SELECT i.ad_id, i.publisher, i.currency, DATE_TRUNC('week', i.date) interval_start, SUM(i.spend) AS spend, SUM(i.impressions) AS impressions, SUM(i.spend) * 10 / NULLIF(SUM(i.impressions::decimal), 0) AS cpm 
   FROM organization_insights i JOIN order_column_trend oct ON i.ad_id = oct.ad_id AND i.publisher = oct.publisher AND i.currency = oct.currency
   WHERE i.date >= DATE_TRUNC('week', CURRENT_DATE - INTERVAL '3 week')
```

Comment:
```suggestion
  WHERE i.date >= DATE_TRUNC('week', CURRENT_DATE - INTERVAL '2 week')
```
I think this should be 2, since this is what we are doing on top.